src/scss/06_components/paragraphs/_react-search.scss

```diff
@@ -98,3 +98,52 @@ fieldset .react-search__checkbox:not(:first-of-type) {
   }
   margin-top: $spacing-double;
 }
+
+.hdbt-search--react {
+  .react-search__form-container {
+    border-bottom: none;
+    @include breakpoint($breakpoint-l) {
+      max-width: 870px;
+    }    
+  }
+
+  .hdbt-search__submit-button {
+    width: 100%;
+
+    @include breakpoint($breakpoint-l) {      
+      padding-left: 130px;
+      padding-right: 130px;
+      width: auto;
+    }
+  }
+
+  .react-search__result-top-area > * {
+    width: 100%;
+
+    @include breakpoint($breakpoint-l) {  
+      width: auto;
+    }
+  }
+  
+  .react-search__results-title {
```

Comment:
Can some of these classes like the new class `results-title` use `hdbt-search--react` prefix e.g. `hdbt-search--react__results-title` because we would like to unify classes we use in react searches. Then we can also use the classes more easier in other searches.

Reply:
Updated